Dockerfile-agent

```diff
@@ -1,25 +1,6 @@
 # arxiv/fulltext-agent
 
-FROM ubuntu:zesty
-
-RUN apt-get update && apt-get install -y \
-    ca-certificates \
-    wget \
-    default-jdk \
-    build-essential \
-    autoconf \
-    libtool \
-    pkg-config \
-    gcc \
-    g++ \
-    libpython3.6 \
-    python3.6 \
-    python3.6-dev \
-    python3.6-venv \
- && rm -rf /var/lib/apt/lists/*
-
-RUN wget https://bootstrap.pypa.io/get-pip.py
-RUN python3.6 get-pip.py
+FROM 626657773168.dkr.ecr.us-east-1.amazonaws.com/arxiv/base:latest
```

Comment:
good practice pushing a consistent source image to a controlled registry, do we feel that we should also pin the tag since latest may be updated?

Reply:
Let's pin after we have an "official" release for the base image.